dev/PowerNotifications/PowerNotifications.idl

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// REVIEW: We're not doing doing any versioning within IDL/WinMDs, right?
+
+namespace Microsoft.ProjectReunion
+{
+    // Enums duplicated from IPowerManagerStatics
+    enum EnergySaverStatus
+    {
+        Disabled = 0,
+        Off,
+        On
+    };
+
+    enum BatteryStatus
+    {
+        NotPresent = 0,
+        Discharging,
+        Idle,
+        Charging
+    };
+    
+    enum PowerSupplyStatus
+    {
+        NotPresent = 0,
+        Inadequate,
+        Adequate
+    };
+
+    // Enums based on on PowerSettingRegisterNotification's state
+    enum PowerSourceStatus
+    {
+        AC = 0,
+        DC,
+        ShortTerm
+    };
+    
+    enum DisplayStatus
+    {
+        Off = 0,
+        On,
+        Dimmed
+    };
+    
+    enum SystemIdleStatus
```

Comment:
As we recently discussed, SystemIdleStatus isn't persistently available, we just get a dataless event pulse when the system is non-idle. So this will unfortunately need to break from the existing pattern, and can't be queried.

If you'd prefer to make this change after the initial PR, that should be fine. Just want to call out that adjustments will need to be made.